content/en/docs/setup/independent/setup-ha-etcd-with-kubeadm.md

```diff
@@ -1,18 +1,18 @@
 ---
 reviewers:
 - sig-cluster-lifecycle
-title: Set up a Highly Availabile etcd Cluster With kubeadm
+title: Set up a High Availability etcd cluster with kubeadm
```

Comment:
@mauilion 👋 For future reference: "High Availability" is lower case: _high availability_, _highly available_. See other uses in this article for examples. 

/cc @Bradamant3